src/fetch/orderbook.py

```diff
@@ -75,7 +75,7 @@ def get_latest_block(cls) -> int:
         return min(int(barn["latest"][0]), int(prod["latest"][0])) - REORG_THRESHOLD
 
     @classmethod
-    def get_orderbook_rewards(cls, block_range: BlockRange) -> DataFrame:
+    def get_order_rewards(cls, block_range: BlockRange) -> DataFrame:
```

Comment:
method comment should probably be updated to reflect that this is the old reward scheme.